primitives/src/schnorr_dsa.rs

```diff
@@ -118,6 +118,7 @@ impl<P: Parameters + Clone> VerKey<P> {
 
 /// Signature secret key pair used to sign messages
 // make sure sk can be zeroized
+#[tagged_blob("SIGNKEY")]
```

Comment:
Should the tag be `SIGNKEYPAIR` just in case we may need to tag the real `SIGNKEY` in future?

Reply:
Sure, I'll change it